client/allocrunner/alloc_runner.go

```diff
@@ -13,7 +12,10 @@ import (
 	"github.com/boltdb/bolt"
 	"github.com/hashicorp/go-multierror"
 	"github.com/hashicorp/nomad/client/allocdir"
+	"github.com/hashicorp/nomad/client/allocrunner/taskrunner"
 	"github.com/hashicorp/nomad/client/config"
+	consulApi "github.com/hashicorp/nomad/client/consul"
```

Comment:
[brainstorming] If we were feeling brave and wanted to prepare for a service-discovery plugins future you could rename this package "discoapi" or similar. Other than the package name there's nothing Consul-specific about it: the methods and structs should be sufficient for any discovery backend.